src/main/java/com/iexec/sms/secret/app/ApplicationRuntimeSecretController.java

```diff
@@ -58,4 +58,16 @@ public ApplicationRuntimeSecretController(AuthorizationService authorizationServ
         return ResponseEntity.noContent().build();
     }
 
+    @RequestMapping(method = RequestMethod.HEAD, path = "/{appAddress}/secrets/{secretIndex}")
+    public ResponseEntity<Void> isApplicationRuntimeSecretPresent(@PathVariable String appAddress,
+                                                                  @PathVariable long secretIndex) {
+        boolean secretExists = applicationRuntimeSecretService.isSecretPresent(appAddress, secretIndex);
```

Comment:
boolean is useless (and method is clearly understandable so no need of tmp variable)

Reply:
Sure, thanks!